rimage/transform/brown_conrady.go

```diff
@@ -11,6 +11,14 @@ type BrownConrady struct {
 	TangentialP2 float64 `json:"tp2"`
 }
 
+// CheckValid checks if the fields for BrownConrady have valid inputs.
+func (bc *BrownConrady) CheckValid() error {
```

Comment:
[opt] might rename this to `CheckExists()` as CheckValid to me implies a check of parameter's presence AND their validity

Reply:
I think this function should also potentially do checks on the values themselves, though for BrownConrady, it isn't so necessary. But the general interface, for all Distorters, should also potentially check the values